tests/tests/cpld_control_nsst

```diff
@@ -0,0 +1,64 @@
+#
+#  cpld_control_nsst test
+#
+
+export TEST_DESCR="Fully coupled FV3-CCPP-MOM6-CICE-CMEPS system - C96MX100"
+
+export CNTL_DIR="cpld_control_nsst"
+
+export LIST_FILES="phyf024.tile1.nc \
+                   phyf024.tile2.nc \
+                   phyf024.tile3.nc \
+                   phyf024.tile4.nc \
+                   phyf024.tile5.nc \
+                   phyf024.tile6.nc \
+                   dynf024.tile1.nc \
+                   dynf024.tile2.nc \
+                   dynf024.tile3.nc \
+                   dynf024.tile4.nc \
+                   dynf024.tile5.nc \
+                   dynf024.tile6.nc \
+                   RESTART/coupler.res \
+                   RESTART/fv_core.res.nc \
+                   RESTART/fv_core.res.tile1.nc \
+                   RESTART/fv_core.res.tile2.nc \
+                   RESTART/fv_core.res.tile3.nc \
+                   RESTART/fv_core.res.tile4.nc \
+                   RESTART/fv_core.res.tile5.nc \
+                   RESTART/fv_core.res.tile6.nc \
+                   RESTART/fv_srf_wnd.res.tile1.nc \
+                   RESTART/fv_srf_wnd.res.tile2.nc \
+                   RESTART/fv_srf_wnd.res.tile3.nc \
+                   RESTART/fv_srf_wnd.res.tile4.nc \
+                   RESTART/fv_srf_wnd.res.tile5.nc \
+                   RESTART/fv_srf_wnd.res.tile6.nc \
+                   RESTART/fv_tracer.res.tile1.nc \
+                   RESTART/fv_tracer.res.tile2.nc \
+                   RESTART/fv_tracer.res.tile3.nc \
+                   RESTART/fv_tracer.res.tile4.nc \
+                   RESTART/fv_tracer.res.tile5.nc \
+                   RESTART/fv_tracer.res.tile6.nc \
+                   RESTART/phy_data.tile1.nc \
+                   RESTART/phy_data.tile2.nc \
+                   RESTART/phy_data.tile3.nc \
+                   RESTART/phy_data.tile4.nc \
+                   RESTART/phy_data.tile5.nc \
+                   RESTART/phy_data.tile6.nc \
+                   RESTART/sfc_data.tile1.nc \
```

Comment:
I don't think this test `cpld_control_nsst` is used? I don't see `SUITE_NAME` being set, how is this test supposed to run/

Reply:
I think you should remove this file `tests/tests/cpld_control_nsst`

Reply:
@XuLi-NOAA You need to either remove this file from the PR or add it as a test in rt.conf. If you add it as a test, you need to set the SUITE_NAME in the test, otherwise it will use the default suite in input.mom6_ccpp.nml.IN

Reply:
> I don't think this test `cpld_control_nsst` is used? I don't see `SUITE_NAME` being set, how is this test supposed to run/

@climbfuji : cpld_control_nsst is not used since it doesn't use v16 physics, as suggested by Bin & Jun, another one, has been created and tested, cpld_bmarkfrac_v16_nsst.

Reply:
Thanks, in this case, please remove `tests/tests/cpld_control_nsst`

Reply:
> Thanks, in this case, please remove `tests/tests/cpld_control_nsst`

@climbfuji : I will remove cpld_control_nsst and it doesn't affect anything we have done.

Reply:
@climbfuji : cpld_control_nsst removed